src/components/base/Button.js

```diff
@@ -0,0 +1,28 @@
+import classNames from 'classnames'
+import Icon from './Icon';
+
+export default function Button({ children ,className, highlight, elevated, type, iconSize, scale }){
+
+  const finalClassnames = classNames('rounded-full p-2 text-white/70 hover:text-white transition-all', className, {
```

Comment:
az once de dedigim gibi base component'te default class olmamasi lazim.........

Reply:
`classNames`  methoduna eklediğim className propu aslında bu default classları overwrite edebiliyo sayfanın stillendirmesi genel olarak belli olduğu için ve gördüğüm butonların hepsindeki genel stillendirmeler böyle olduğu içim mantıklı diye düşündüm her şekilde overwrite edebiliyorum beğenmezsem diye. çünkü genel yapıda butonları hepsinin stili buyken her kullandığımda aynı stilleri yazıp yazıp duracağım öbür türlü

Reply:
baska bir seyle wraplemen bence daha mantikli gibime geliyor